Jenkinsfile

```diff
@@ -61,10 +61,7 @@ def build(python_version, compiler, label, release=false) {
 
                     CC=${compiler} ${VENV}/bin/python setup.py sdist bdist_wheel
                     ${VENV}/bin/python ${VENV}/bin/pip install -e .
-                    ${VENV}/bin/python ${VENV}/bin/nosetests \
-                        --with-xunit --with-coverage --cover-inclusive \
-                        --cover-package=anonlink
-
+                    ${VENV}/bin/python ${VENV}/bin/pytest --cov=anonlink
```

Comment:
If you add something like `--junit-xml=testout.xml --cov-report=xml:coverage.xml` we can then pick up the report/s with jenkins.